docs/UserGuide.md

```diff
@@ -207,17 +210,20 @@ Edits an existing lend or borrow in the record. After a successful edit, the upd
 - Cannot change a `lend` record to a `borrow` record or vice versa.
 - Our application does not support input names with a slash '/'.
 - The input year cannot be earlier than 2000, and the maximum year that can be set is 9999.
+- Return date should be equal to or later than the present date.
+- Borrow date must be before the return date.
+- All parameters must be present in this command.
 
 **Examples:**
 
-- `edit 17 d/2023-02-02 n/Akshay Narayan a/25.10 b/2023-04-02 p/CS2040`
+- `edit d/2022-02-02 n/Akshay Narayan a/25.10 b/2024-07-14 p/CS2113`
```

Comment:
Good catch for the deadline date, but also you left out the index for the command